examples/tutorials/RecommendersSVD.md

```diff
@@ -0,0 +1,13 @@
+# Automatically tuning SVD on NNI
```

Comment:
change the name of tutorials?

Reply:
"practice" is also a good choice, looking for others' opinion.